index.html

```diff
@@ -4,24 +4,37 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
+    <title>Blessing's Portfolio</title>
     <link rel="stylesheet" href="styles.css">
     <link href='https://fonts.googleapis.com/css?family=Crete Round' rel='stylesheet'>
     <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
     <link href='https://fonts.googleapis.com/css?family=Inter' rel='stylesheet'>
+    
 </head>
 <body>  
+    <section class="section">
     <main class="main">
         <div class="divNav">   
             <nav class="headerNav">             
                     <img class="toggleIcon" src="images/toggleIcon.png" alt="toggle">                                      
-                    <h3 class="logo">John Doe </h3> 
-                    <ul class="dskNav" >
-                        <li>Portfolio</li>
-                        <li>About</li>
-                        <li>Contact</li>
-                        <li><img src="images/Envelop.png" alt="Envelop"></li>
-                    </ul>                                       
+                    <h3 class="logo">Okpara Blessing </h3>
+                    <div id="show">
+                        <ul id="style"   >
+                            <div id="cross-icon">
+                                <img onclick="hide()" class="style-li" src="/images/close-Icon.png" alt="close-Icon">
+                              </div>
```

Comment:
- [ ] Kindly eliminate the horizontal scroll in your website which results in this excess white space appearing on the right for your project to be more professional.
![image](https://user-images.githubusercontent.com/62290545/203624685-bb5d8c18-0394-42ff-81cd-505c90aec350.png)